submissions/strumerlz/html-css-popup/index.html

```diff
@@ -0,0 +1,160 @@
+                                        <span>Calendar</span>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                        <a href="#">
+                                        <img class="submenu-icon" src="assets/submenu_icons/Contacts.png" alt="Contacts">
+                                        <span>Contacts</span>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                        <a href="#">
+                                        <img class="submenu-icon" src="assets/submenu_icons/photos.png" alt="Photos">
+                                        <span>Photos</span>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                        <a href="#">
+                                        <img class="submenu-icon" src="assets/submenu_icons/video.png" alt="Video">
+                                        <span>Video</span>
+                                        </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                    <a href="#">
+                                        <img class="submenu-icon" src="assets/submenu_icons/music.png" alt="Music">
+                                        <span>Music</span>
+                                    </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                    <a href="#">
+                                        <img class="submenu-icon" src="assets/submenu_icons/podcast.png" alt="Podcasts">
+                                        <span>Podcasts</span>
+                                    </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                    <a href="#">
+                                        <img class="submenu-icon" src="assets/submenu_icons/map.png" alt="Map">
+                                        <span>Map</span>
+                                    </a>
+                                    </li>
+                                    <li class="submenu-item">
+                                    <a href="#">
+                                        <img class="submenu-icon" src="assets/submenu_icons/idea.png" alt="Ideas">
+                                        <span>Ideas</span>
+                                    </a>
+                                    </li>
+                                </ul>
+                                
```

Comment:
could you please remove spaces (empty lines)? would be better to use code formatting (looks like Prettier, etc, it depends on what you use)

Reply:
Thanks for advice! Corrected it.